benchmark/CMakeLists.txt

```diff
@@ -61,6 +61,8 @@ endif()
 
 if(TODBC_WINDOWS)
     set_source_files_properties(tb.c PROPERTIES COMPILE_OPTIONS "/source-charset:utf-8")
+else()
+    target_link_libraries(tb pthread)
```

Comment:
is this necessary?

Reply:
Yes, I added it because of a compilation error, and assume that thread library is automatically added in some Linux
 releases, but this is not standard behavior